DToCToDB.md

```diff
@@ -0,0 +1,1193 @@
+    Int64Index: 176060 entries, 0 to 15919
+    Data columns (total 16 columns):
+    Year                              176060 non-null int64
+    Month                             176060 non-null int64
+    Provider Parent Org Code          176060 non-null object
+    Provider Parent Name              176060 non-null object
+    Provider Org Code                 176060 non-null object
+    Provider Org Name                 176060 non-null object
+    Local Authority Code              176060 non-null object
+    Local Authority Name              176060 non-null object
+    Acute Or Non Acute Description    176060 non-null object
+    Reason For Delay                  176060 non-null object
+    NHS A SUM                         176051 non-null float64
+    NHS B SUM                         176044 non-null float64
+    Social Care A SUM                 176052 non-null float64
+    Social Care B SUM                 176048 non-null float64
+    Both A SUM                        176060 non-null int64
+    Both B SUM                        176059 non-null float64
+    dtypes: float64(5), int64(3), object(8)
+    memory usage: 22.8+ MB
+
+
+Combine all of these dataframes into a single dataframe, ready for upload to SQLite Database.
+
+
+```python
+frames = [dtoc_16, dtoc_15, dtoc_14, dtoc_13, dtoc_12]
+dtoc = pd.concat(frames)
+
+dtoc.info()
+```
+
+    <class 'pandas.core.frame.DataFrame'>
+    Int64Index: 921880 entries, 0 to 15919
+    Data columns (total 16 columns):
+    Year                              921880 non-null int64
+    Month                             921880 non-null int64
+    Provider Parent Org Code          921880 non-null object
+    Provider Parent Name              921880 non-null object
+    Provider Org Code                 921880 non-null object
+    Provider Org Name                 921880 non-null object
+    Local Authority Code              921880 non-null object
+    Local Authority Name              921880 non-null object
+    Acute Or Non Acute Description    921880 non-null object
+    Reason For Delay                  921880 non-null object
+    NHS A SUM                         921843 non-null float64
+    NHS B SUM                         921825 non-null float64
+    Social Care A SUM                 921861 non-null float64
+    Social Care B SUM                 921848 non-null float64
+    Both A SUM                        921872 non-null float64
+    Both B SUM                        921874 non-null float64
+    dtypes: float64(6), int64(2), object(8)
+    memory usage: 119.6+ MB
+
+
+As noted above, we have a number of null values. There are two approches to dealing with this problem:
+* Drop the observation (row) entirely
+* Replace the null with a value
+
+In this case as we are dealing with numeric (float) values, therefore we are going to replace the null value with 0. So that other values in that observation are not lost. This is achieved using the `df.fillna()` function:
```

Comment:
The site you got the data from might have more information about missing values, and what they mean, which could help you make a decision